thing/index.js

```diff
@@ -477,15 +477,19 @@ function ThingShadowsClient(deviceOptions, thingShadowOptions) {
             discardStale: true,
             enableVersioning: true,
             qos: 0,
-            pending: false
+            pending: false,
+            pendingDeltaSubscribe: true,
```

Comment:
If this flag is pending, we should probably fail any shadow requests. We might be able to reuse the pending flag for the same purpose?